lib/ht/search_client/test.rb

```diff
@@ -38,11 +38,11 @@ def stay_class?
       self.class.to_s =~ /Stay/
     end
 
-    def mocked_results(properties)
+    def mocked_results(properties, total)
       {
         page:     params.fetch(:page),
         per_page: params.fetch(:per_page),
-        total:    properties.length,
+        total:    total,
```

Comment:
Remember that the idea is *"mocking"* not validating things - I agree that before there was no control over the `total` which might be interesting to test pagination in a view for example hence be able to control that value. That's why my suggestion: by default try to be nice, if you supply something then you have to guarantee all is good.  

If you really want to add validation then I think we should raise a error if you try to do the above.

My 2 cents.